app/code/core/Mage/Adminhtml/Block/Sales/Order/Grid.php

```diff
@@ -244,4 +245,29 @@ public function getGridUrl()
     {
         return $this->getUrl('*/*/grid', ['_current' => true]);
     }
+
+    public function decorateStatus($value, $row, $column, $isExport): string
+    {
+        $colors = Mage::registry('order_status_colors');
```

Comment:
Could we alternatively use a protected variable `$this->orderStatusColors` since it's only being used in this block?

It's actually really not a big deal either way, just a thought.

Reply:
done ;-)